content/en/security_platform/explorer.md

```diff
@@ -57,6 +57,21 @@ Below the overview of the signal are tabs with detailed information related to t
 
 - `Related Issues` includes a list of other signals which contain the same group by values to assist with triaging the signal.
 
+### Threat Intelligence
+
+Datadog Security Monitoring offers threat intelligence feeds curated by threat intelligence partners. These feeds are constantly updated to include data about known suspicious activity (for example, IOCs), so you can quickly identify which potential threats to address.
+
+{{< img src="security_platform/security_monitoring/explorer/threat_intelligence.png" alt="Threat Intelligence in the Security Signals Explorer"  >}}
```

Comment:
This image looks like it needs some blurring or cropping of sensitive info

Reply:
@jmassey-datadog Does this image contain sensitive info? It's being used in other places throughout the docs / blog.